### PR TITLE
PZ-3290 PayPal mandate creation via API

### PR DESCRIPTION
**Pull request by MollieRick**

### CODE REVIEW

**Comment by MollieRick** [comment]
TODO; changelog

File: source/reference/v2/mandates-api/create-mandate.rst [hunk 1]
```diff
@@ -72,6 +74,24 @@ Replace ``customerId`` in the endpoint URL by the customer's ID, for example ``/
      - A custom mandate reference. Use an unique ``mandateReference`` as some banks decline a
        Direct Debit payment if the ``mandateReference`` is not unique.
 
+   * - ``paypalBillingAgreementId``
+
+       .. type:: string
+          :required: false
+
+     - The billing agreement ID given by PayPal.
```
Comment (willemstuursma) [inline source/reference/v2/mandates-api/create-mandate.rst:82]: Could use an example.

Comment (MollieRick) [reply source/reference/v2/mandates-api/create-mandate.rst:91]: As in?

Comment (ndijkstra) [reply source/reference/v2/mandates-api/create-mandate.rst:91]: He means something like `B-1425456412` (if this is the correct format)

File: source/reference/v2/mandates-api/create-mandate.rst [hunk 2]
```diff
@@ -12,8 +12,8 @@ Create mandate
    :organization_access_tokens: true
    :oauth: true
 
-Create a mandate for a specific customer. Mandates allow you to charge a customer's credit card or bank account
-recurrently.
+Create a mandate for a specific customer. Mandates allow you to charge a customer's credit card,
+PayPal account or bank account recurrently.
 
 It is only possible to create mandates for IBANs with this endpoint. To create mandates for credit cards, have your
 customers perform a :ref:`'first payment' <payments/recurring/first-payment>` with their credit card.
```
Comment (ndijkstra) [inline source/reference/v2/mandates-api/create-mandate.rst:19]: You need to add PayPal here

Comment (MollieRick) [reply source/reference/v2/mandates-api/create-mandate.rst:19]: Its in there? ;p

**Pull request merged**